### PR TITLE
Clean up the console app for broader consumption

### PR DESCRIPTION
**Pull request by jasonleenaylor**
* Package the console app as a dotnet tool
* Eliminate the Windows.Forms dependency
* Improve the argument handling (stop short of using an args lib)
* Support several sensible frameworks

### CODE REVIEW

File: IdlImpTool/IDLImp.cs [hunk 1]
```diff
@@ -6,7 +6,8 @@
 using System.Collections.Generic;
 using System.Collections.Specialized;
 using System.IO;
-using System.Windows.Forms;
+using System.Net.Mime;
```
Comment (ermshiperete) [inline IdlImpTool/IDLImp.cs:9]: I'm surprised that `System.Net.Mime` namespace is needed

File: IdlImpTool/IDLImp.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```
Comment (ermshiperete) [inline IdlImpTool/IDLImp.csproj:4]: Why `net5` and not `net6`? At this point only `netcoreapp3.1` is still in support...

Comment (jasonleenaylor) [reply IdlImpTool/IDLImp.csproj]: Because I didn't have VS2022 on my machine yet. :( And neither does Appveyor..

Comment (jasonleenaylor) [reply IdlImpTool/IDLImp.csproj:4]: https://github.com/sillsdev/IdlImporter/issues/22

**Pull request merged**